onchain/rollups/test/foundry/dapp/CartesiDApp.t.sol

```diff
@@ -42,14 +42,15 @@ contract CartesiDAppTest is TestBase {
     error UnexpectedOutputEnum(
         LibServerManager.OutputEnum expected,
         LibServerManager.OutputEnum obtained,
-        uint256 inputIndex
+        uint256 inputIndexWithinEpoch
     );
 
-    error InputIndexOutOfBounds(uint256 length, uint256 inputIndex);
+    // IIWE stands for InputIndexWithinEpoch
+    error IIWEOutOfBounds(uint256 length, uint256 inputIndexWithinEpoch);
```

Comment:
If a developer (not too familiar with the code) runs `forge test` and gets this error, they will have no idea what `IIWE` means. I would, therefore, not abbreviate this error name.
```suggestion
    error InputIndexWithinEpochOutOfBounds(uint256 length, uint256 inputIndexWithinEpoch);
```